packages/project-access/src/library/types.ts

```diff
@@ -0,0 +1,24 @@
+export const enum ReuseLibType {
+    Library = 'library',
+    Component = 'component'
+}
+
+/**
+ * Reuse library
+ */
+export interface ReuseLib {
+    name: string;
+    path: string;
+    type: ReuseLibType;
+    uri: string;
+    dependencies: string[];
+    libRoot: string;
+}
+
+/**
+ * Choice for the reuse library
+ */
+export interface ReuseLibChoice {
```

Comment:
Prompting related types should be in the prompting module. Also should extend  the inquirer type `extends ListChoiceOptions` or which-ever is correct

Reply:
Moved as requested